src/pages/MyKiva/MyKivaPage.vue

```diff
@@ -14,9 +20,32 @@ import MyKivaPageContent from '#src/pages/MyKiva/MyKivaPageContent';
  */
 export default {
 	name: 'MyKivaPage',
+	inject: ['apollo', 'cookieStore'],
```

Comment:
Do we need `cookieStore`?

Reply:
Looks like that, otherwise we get an error
`Error: Missing required injections! Add "inject: ['apollo', 'cookieStore']" to this component definition`